reference/profiles.rst

```diff
@@ -217,6 +244,68 @@ The ``include()`` statement has to be at the top of the profile file:
     zlib:CC=/usr/bin/clang
     zlib:CXX=/usr/bin/clang++
 
+
+.. _profiles_configuration_data_operators:
+
+Configuration data operators
++++++++++++++++++++++++++++++++++
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+It's also possible to use some extra operators when you're composing different tools configurations:
+
+.. code-block:: text
+    :caption: global.conf
+
+    [settings]
+    ...
+
+    [conf]
+    # Defining several lists
+    user.myconf.build:ldflags=["-mmacosx-version-min"]
```

Comment:
I think that using `-mmacosx-version-min` that is passed by the XcodeToolchain depending of the value of `os.version` or `-arch`, `-isysroot` flags make this example a bit confusing... could we use another flags or another type of conf for the example?

Reply:
If they could be confusing then I think that I cannot use any kind of real flags because it'll be confusing as well. I'll use some fake ones only to demonstrate the lists part

Reply:
Yes, please, use fake values, not real ones, unless it is a real example of something that really works.